guilford_court_commitment.yaml

```diff
@@ -0,0 +1,408 @@
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+    name: "jailstays"
+
+  jailtimemodifications:
+    fqn: "ol.jailtimemodifications"
+    entitySetName: "GuilfordJailTimeCredit"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "TOTAL JAIL CREDITS (IN DAYS)"
+      ol.value: # total jail credit in days
+        type: "ol.value"
+        column: "TOTAL JAIL CREDITS (IN DAYS)"
+      ol.units: # value transform and write in days
+        type: "ol.units"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Days"
+    name: "jailtimemodifications"
+
+  charges:
+    fqn: "ol.charge"
+    entitySetName: "GuilfordCharges"
+    propertyDefinitions:
+      ol.id: # most serious offense code
+        type: "ol.id"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "MOST SERIOUS OFFENSE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "MOST SERIOUS OFFENSE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      ol.description: # most serious offense code
+        type: "ol.description"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "MOST SERIOUS OFFENSE CODE"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "MOST SERIOUS OFFENSE CODE"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+    name: "charges"
+
+  supervisions:
+    fqn: "ol.supervision"
+    entitySetName: "GuilfordSupervisions"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        transforms:
+        - !<transforms.ConcatTransform>
+          columns: ["P\  CASE INTAKE DATE", "OFFENDER NC DOC ID NUMBER"]
+      ol.recognizedstartdate: # p&p commitment status flag
+        type: "ol.recognizedstartdate"
+        column: "P&P CASE INTAKE DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.status: # p&p commitment status flag
+        type: "ol.status"
+        transforms:
+        - !<transforms.BooleanRegexTransform>
+          column: "P&P COMMITMENT STATUS FLAG"
+          pattern: "\\?+"
+          transformsIfFalse:
+          - !<transforms.ColumnTransform>
+            column: "P&P COMMITMENT STATUS FLAG"
+          transformsIfTrue:
+          - !<transforms.ValueTransform>
+            value: ""
+      general.datetime: # p&p commitment status date
+        type: "general.datetime"
+        column: "P&P COMMITMENT STATUS DATE"
+        transforms:
+        - !<transforms.DateTimeTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/New_York"
+      ol.durationdays: # total length of supervision
```

Comment:
this is also a double, please double check that it integrated correctly without needing a transformation.